pkg/storage/uploader.go

```diff
@@ -95,7 +101,18 @@ func (u *FileUploader) Upload(ctx context.Context) error {
 
 	fmt.Println("Upload successful :", cid)
 
-	err = u.DBClient.CreateJob(ctx, cid.String(), fname)
+	var timestamp *int64
+	if _, ok := metadata["timestamp"]; !ok {
+		fmt.Println("timestamp is missing", fname)
+	} else {
+		t, err := strconv.ParseInt(metadata["timestamp"], 10, 64)
+		if err != nil {
+			return fmt.Errorf("failed to parse timestamp: %v", err)
+		}
+		timestamp = &t
```

Comment:
why `timestamp` is `*int64` not `int64`?

Reply:
it's just to represent with `nil` when `timestamp` is missing, it shouldn't be missing. but if we deploy the CF first, it will actually be missing